src/sql/workbench/contrib/notebook/browser/outputs/renderers.ts

```diff
@@ -338,10 +338,7 @@ export function renderSVG(options: renderSVG.IRenderOptions): Promise<void> {
 		return Promise.resolve(undefined);
 	}
 
-	// Render in img so that user can save it easily
-	const img = new Image();
-	img.src = `data:image/svg+xml,${encodeURIComponent(source)}`;
-	host.appendChild(img);
+	host.innerHTML = source;
```

Comment:
What about the comment in the original code about being able to "save it easily"? Do you know what that meant?